test/common/src/main/kotlin/org/codice/compliance/utils/TestCommon.kt

```diff
@@ -42,42 +36,26 @@ class TestCommon {
                 "ItMustNotExceed80BytesInLength"
         const val MAX_RELAYSTATE_LEN = 80
 
-        val idpMetadata = Common.parseIdpMetadata()
+        private val DEPLOY_CL = getDeployDirClassloader()
         private val spMetadata = Common.parseSpMetadata()
 
+        val idpMetadata = Common.parseIdpMetadata()
         val SP_ISSUER = spMetadata.keys.first()
-        private val SP_INFO = spMetadata[SP_ISSUER]
-
-        var ACS_URL = SP_INFO?.getAssertionConsumerService(SamlProtocol.Binding.HTTP_REDIRECT)?.url
-
-        private val DEPLOY_CL = getDeployDirClassloader()
+        var acsUrl : String? = ""
 
         /**
-         * Generates and returns a POST Authn Request
+         * Returns the Assertion Consumer Service URL for the provided binding
          */
-        fun generateAndRetrieveAuthnRequest(): String {
-            OpenSAMLUtil.initSamlEngine()
```

Comment:
❓ Are we sure we don't need this anymore?

Reply:
We moved the generation of the request to the tests themselves. We are already using an AuthnRequest builder so tests should have the "choice" on what fields to provide. I deleted the method but the logic is in the tests.

Reply:
Sorry for not being more clear... I meant this `initSamlEngine` method call. I don't see it being moved to somewhere else. It's just removed.

Reply:
Oh, I see. Everything seemed like it was working without it. I'm honestly not sure what that does